js/views/standalone_registration_view.js

```diff
@@ -50,14 +53,40 @@
       'change #code': 'onChangeCode',
       'click #register': 'registerWithoutMnemonic',
       'click #register-mnemonic': 'registerWithMnemonic',
+      'click #back-button': 'onBack',
+      'click #save-button': 'onSaveProfile',
       'change #mnemonic': 'onChangeMnemonic',
       'click #generate-mnemonic': 'onGenerateMnemonic',
       'change #mnemonic-display-language': 'onGenerateMnemonic',
       'click #copy-mnemonic': 'onCopyMnemonic',
       'click .section-toggle': 'toggleSection',
-      'keyup #password': 'onPasswordChange',
+      'keyup #password': 'onValidatePassword',
       'keyup #password-confirmation': 'onValidatePassword',
     },
+    async showPage(pageIndex) {
+      // eslint-disable-next-line func-names
+      this.$pages.each(function(index) {
```

Comment:
Is there a reason this isn't using arrow function?

Reply:
`$(this)` is required and using the arrow function will bind `this` to the registration view instead making it impossible to access the page instance